src/components/rangeFilterInput/__tests__/Index.Test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen, waitFor } from '@testing-library/react';
+
+import RangeFilterInput from '../index';
+
+describe('<RangeFilterInput/>', () => {
+  const mockOnChange = jest.fn();
+
+  const renderInputField = () => {
+    return render(
+      <RangeFilterInput
+        handleChange={mockOnChange}
+        name={{
+          from: 'priceFrom',
+          to: 'priceTo',
+        }}
+        unit="CHF"
+        value={{
+          from: 200,
+          to: 1000,
+        }}
+        placeholder={{
+          from: 'From',
+          to: 'To',
+        }}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    mockOnChange.mockClear();
+  });
+
+  it('triggers onChange with the touched FROM field', async () => {
+    renderInputField();
+    const inputFrom = screen.getByPlaceholderText('From');
+
+    userEvent.clear(inputFrom);
+    userEvent.type(inputFrom, '500');
```

Comment:
These interactions change component state. Please wrap them in `act1
```suggestion
    await act(() => userEvent.clear(inputFrom));
    await act(() => userEvent.type(inputFrom, '500'));
```

Reply:
I also applies to the `to` field test